cmd/libsnap-confine-private/cgroup-support.h

```diff
@@ -37,4 +37,24 @@ void sc_cgroup_create_and_join(const char *parent, const char *name, pid_t pid);
  **/
 bool sc_cgroup_is_v2(void);
 
+/**
+ * sc_cgroup_mount_snapd_hierarchy mounts /run/snapd/cgroup if one is missing.
+ *
+ * The logic mounts an v1 cgroup hierarchy with name=snapd and without any
+ * controllers. Currently no release agent is set and no release notification
+ * is enabled.
+ **/
+void sc_cgroup_mount_snapd_hierarchy(void);
+
+/**
+ * Join the name=snapd cgroup for the given snap.
+ *
+ * This function adds the specified process to the named cgroup named after the
+ * snap security tag.
+ *
+ * This hierarchy is designed for tracking processes and associating them with
+ * a given executable portion of a snap (either an application or a hook).
```

Comment:
This last sentence makes more sense in describing sc_cgroup_mount_snapd_hierarchy().